src/Pos/Wallet/Web/Server/Full.hs

```diff
@@ -1,31 +1,43 @@
+{-# LANGUAGE DataKinds           #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeOperators       #-}
 
 -- | Module for full-node implementation of Daedalus API
 
 module Pos.Wallet.Web.Server.Full
        ( walletServeWebFull
+       , walletServeWebFullS
        , walletServerOuts
+       , liftWMode
+       , runWStatsMode
+       , runWProductionMode
+       , runWStaticMode
+       , WalletStatsMode
+       , WalletProductionMode
+       , WalletStaticMode
        ) where
 
-import           Universum
```

Comment:
No, I meant `serious`. Why did you move it? https://github.com/serokell/serokell-util/blob/master/serokell-style.md#imports